middleware/lib/claim-client.js

```diff
@@ -100,6 +97,34 @@ class ClaimClient {
     });
   }
 
+    /**
+   * Unarchive a single file from a claim.
+   *
+   * @param {string} vin  - The VIN of the associated claim.
+   * @param {string} name - The name of the file to unarchive.
+   */
+  async unarchiveFile(vin, name) {
```

Comment:
I'd prefer to just modify the 1 line of `archiveFile` and add a flag argument to make this work for both archiving and unarchiving. The code is otherwise essentially the exact same. 

It'll be easier to maintain that way.